pkgs/development/libraries/libxml2/default.nix

```diff
@@ -53,6 +53,12 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  # disable test that's problematic with newer pythons: see
+  # https://mail.gnome.org/archives/xml/2017-August/msg00014.html
+  preCheck = lib.optionalString (pythonSupport && !(python?pythonOlder && python.pythonOlder "3.5")) ''
```

Comment:
`python?pythonOlder` did you notice an interpreter that did not have this? A common `passthru` is used for all Python interpreters.

Reply:
Meh I was just fitting in with lower down where `python?isPy3` is used.